demo/src/main/AndroidManifest.xml

```diff
@@ -20,6 +19,10 @@
       </intent-filter>
     </activity>
 
+    <meta-data
+      android:name="com.novoda.downloadmanager.MaxConcurrentDownloads"
+      android:value="1" />
```

Comment:
you can't make the demo just do a single download. Make two demo apps? What about all the people who wanna see multiple downloads  ... or set it with flavors, I dunno just need to demo both to the user :x: